### PR TITLE
Add nmn bond prereqs to 2.5.10 docs

### PR DESCRIPTION
**Pull request by keopp-hpe**
#### Summary and Scope
This PR adds prereqs for NMN bonding to the documentation in 2.5.10.

### CODE REVIEW

File: docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md [hunk 1]
```diff
@@ -11,6 +11,13 @@ In the command examples in this procedure, `PRODUCT_VERSION` refers to the curre
 
 By default, the Node Management Network \(NMN\) is connected to a single `nmn0` interface.  If desired, and the system networking is configured to support it, the Node Management Network may be configured as a bonded interface, `nmnb0`. To configure the NMN as a bonded pair, set `uan_nmn_bond` to true and set the interfaces to be used in the bond in `uan_nmn_bond_slaves` as described in [UAN Ansible Roles](UAN_Ansible_Roles.md).
 
+**Prerequisites for Bonded NMN:**
+
+To enable NMN bonding, certain features in iPXE and the management switches connected to the UAN NMN interfaces must be enabled.  These features are enabled in the following CSM and CANU versions:
+
+* CSM-1.3.2, CSM-1.4 and newer enable the iPXE features
```
Comment (alanm-hpe) [inline docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md:18]:
```suggestion
* CSM-1.3.2 and newer enable the iPXE features
```

File: docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md [hunk 2]
```diff
@@ -11,6 +11,13 @@ In the command examples in this procedure, `PRODUCT_VERSION` refers to the curre
 
 By default, the Node Management Network \(NMN\) is connected to a single `nmn0` interface.  If desired, and the system networking is configured to support it, the Node Management Network may be configured as a bonded interface, `nmnb0`. To configure the NMN as a bonded pair, set `uan_nmn_bond` to true and set the interfaces to be used in the bond in `uan_nmn_bond_slaves` as described in [UAN Ansible Roles](UAN_Ansible_Roles.md).
 
+**Prerequisites for Bonded NMN:**
+
+To enable NMN bonding, certain features in iPXE and the management switches connected to the UAN NMN interfaces must be enabled.  These features are enabled in the following CSM and CANU versions:
+
+* CSM-1.3.2 and newer enable the iPXE features
+* CANU-1.7.0 and newer enables the management switch features
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md:19]: Is CANU it's own separate software product? I thought it was part of CSM.

**Review by darrenn-jackson — CHANGES_REQUESTED** [review]
1 formatting change. One question about the CSM and CANU versions

File: docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md (hunk 2 above)
Comment (keopp-hpe) [reply docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md:19]: It's both.  Part in CSM, but then CSM instructs the admin to download the latest version from GitHub.  So it is noted here what version this appears in.

**Review by alanm-hpe — APPROVED** [review]

**Review by darrenn-jackson — APPROVED** [review]

**Pull request merged**